x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/ClientTransformIndexer.java

```diff
@@ -314,7 +320,7 @@ protected void persistState(TransformState state, ActionListener<Void> listener)
                     // - this is not a catastrophic failure, if 2 state persistence calls run at the same time, 1 should succeed and update
                     // seqNoPrimaryTermAndIndex
                     // - for tests fail(assert), so we can debug the problem
-                    logger.error(
+                    logger.warn(
```

Comment:
We are now treating this as a "catastrophic" failure in that we will fail the transform after 10 failures. 

Which, I am not sure will ever happen because the failure only occurs if one other stats storage succeeded before and incremented the doc seq_no and primary_term.

Reply:
The change _decreases_ the log level from `ERROR` to `WARN` not vice versa.

Reply:
I am meaning the comment above the message.

Reply:
good point, I will update the documentation